upgrade/lab5.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
```

Comment:
Can you add the same check to see passed path to unpacked upgrade tarball like scenarios 2-4 for `lab5.sh` - `lab7.sh`

```
if [ -z "$1" ]; then
    echo "Please pass a path to the unpacked upgrade tarball, for example: ./labX.sh /home/ubuntu/gravity-5.5.50"
    exit 1
fi
```